xml/article_virtualization.xml

```diff
@@ -273,21 +274,21 @@
     Standard real-time dangers apply in that if the intention is to run a
```

Comment:
"Standard real-time dangers apply [...]" sounds a little off...

Reply:
I haven't touched this. It may be overly formal but I think it's clear.

Reply:
I know you haven't touched it, it's white line after all. But I stumbled a bit reading that "dangers apply" -- the noun and the verb very much clash in my head.

Reply:
It's not common, no. Maybe "warnings apply" or "caveats apply" is more usual.

If you want to change it to, for example, "standard warnings about real-time dangers apply", perhaps, then I am fine with that. I try to avoid changes that make things longer, except for the common mistake of identifying a list with "like X" -- which strictly means "things that resemble X, but  excluding X". This I replace with "such as" even though it's longer.

Reply:
I think there's quite a bit of potential to make the sentence much less wordy overall anyway. Suggestion:

`As usual, when running compute-intensive applications with real-time priority, make sure that kernel threads cannot starve.`

Reply:
Accepted, and that is clearer. I incorporated this as a separate commit.